### ISSUE TITLE
My SimpleWall PopUps UI is blank/white and all I can do is click the top-right x to close it!

### ISSUE THREAD

**Issue by blueauran** [henrypp/simplewall#927]
I have been having this issue for some time now...and because this is the best firewall in existence IMO I wish to get it sorted somehow to restore proper functionality.  WHAT is happening is that whenever a security popup comes from the firewall, IT'S ALL BLANK basically and I just have to shut it off by pressing the X in the top right corner.  In fact that is the ONLY thing I can see and do.  It really bothers me because everything seems to start out fine, then this happens after a couple weeks or whatever, and I have tried everything I can think of yet it's still stuck that way for every popup.  So I have my Win10 UI set to dark mode, and I also use that for most of my apps also, and even in my Brave browser.  I also am using Nvidia, and maybe I can change something in that in order to make this work?  So yes, my popup is all white, and all the boxes are missing, even if I click around everywhere it seems like the UI is missing, like I said only thing I can really do is click the X in the top right...anyone have any suggestions for a fix...like maybe update to a beta version, or see if I shut dark mode off if it will fix (I still have to have darkmode yet at least I could test it to see if this is the issue), in fact I will do that NOW and see if it changes anything just out of curiosity...btw this has been happening in all the old versions I used to pretty much, it's weird.  I hope I can get it fixed!  
  Ok, just found out that I had apps set to LIGHT version, so I switched that to Dark properly...I have also changed the font size to 14, and I am pretty sure this is going to pull this off (hopes)...so just a little correction to let you know what I am trying!

**Comment by henrypp** [comment]
tl;dr;

no tech. info

**Comment by blueauran** [comment]
well, changing the app to dark mode made it work again and be visible...so problem solved, and I did it all on my own with zero assistance...which is proper as I have more exp. than anyone here most likely!

**Comment by blueauran** [comment]
Best FREE firewall in existence, used to be that there was one better, yet it's been gone for years now...so this is the new #1 and will be pretty much for the next 5-10 IMO years for windows...

**Comment by blueauran** [comment]
THIS was the old best one, yet they kinda butchered it a bit and it's lacks all the goodies it used to have IMO https://www.binisoft.org/
maybe you can see some of the options in it that you may be able to incorporate into yours...I offer it here in case you missed it.
